_attributes/common-attributes.adoc

```diff
@@ -37,11 +37,33 @@
 :ocp: OpenShift Container Platform
 :olm-first: Operator Lifecycle Manager (OLM)
 :olm: OLM
-:ocp-supported-version: 4.15
-:rhacs-version: 4.4
-:quay-version: 3.11
+// These variable are the current product release versions for the OPP products
+:ocp-supported-version: 4.16
+:rhacs-version: 4.5
+:quay-version: 3.12
 :rhacm-version: 2.10
-:odf-version: 4.15
+:odf-version: 4.16
+// These variables are for the immediate past 4 OPP product versions that are in the compatibility table
+:ocp-supported-version-1: 4.15
+:rhacs-version-1: 4.4
+:quay-version-1: 3.11
+:rhacm-version-1: 2.10
+:odf-version-1: 4.15
+:ocp-supported-version-2: 4.14
+:rhacs-version-2: 4.3
+:quay-version-2: 3.10
+:rhacm-version-2: 2.9
+:odf-version-2: 4.14
+:ocp-supported-version-3: 4.13
+:rhacs-version-3: 4.0
+:quay-version-3: 3.8
+:rhacm-version-3: 2.7
```

Comment:
We should have had ACM 2.8 here.  Can you double check to see when we documented ACM 2.8?

Reply:
@gparvin after looking at the previous PRs to update the versions, 2.8 follows the trend, however, I will have Oded verify this. Thank you!